pkg/kubelet/kuberuntime/kuberuntime_manager.go

```diff
@@ -776,6 +788,16 @@ func (m *kubeGenericRuntimeManager) computePodActions(pod *v1.Pod, podStatus *ku
 					}
 					changes.ContainersToRestart = append(changes.ContainersToRestart, idx)
 					keepCount--
+				} else {
+					if resizeMemLim {
+						markContainerForUpdate("memory-limit", specLim.Memory().Value(), statusLim.Memory().Value())
```

Comment:
naming convention should be followed here. memory-limits as a variable name seems not follow the k8s naming convention.

Reply:
It's a constant string. I'll replace it with declared constants.